packages/ffe-file-upload-react/src/FileItem.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { number, func, shape, object, string as stringType } from 'prop-types';
+
+const FileItem = props => (
```

Comment:
Could you please deconstruct props here?
It's nice to be able to see available props right away, and you'll save yourself alot of `props.` throughout the code.